pkg/cloud/services/compute/instance_test.go

```diff
@@ -48,6 +49,11 @@ func Test_getPortName(t *testing.T) {
 			args: args{"test-1-instance", &infrav1.PortOpts{}, 4},
 			want: "test-1-instance-4",
 		},
+		{
+			name: "with PortOpts name suffix",
+			args: args{"test-1-instance", &infrav1.PortOpts{NameSuffix: "foo2", NetworkID: "bar", DisablePortSecurity: &truePtr}, 4},
```

Comment:
```suggestion
			args: args{"test-1-instance", &infrav1.PortOpts{NameSuffix: "foo2", NetworkID: "bar", DisablePortSecurity: pointer.Bool(true)}, 4},
```

Just a hint, not really a finding: you could use `k8s.io/utils/pointer`

Reply:
ah, that is convenient. Its unusually difficult to declare a bool pointer in go.